source/beamline/AMControl.h

```diff
@@ -633,6 +663,26 @@ protected slots:
 	/*! If isEnum() returns true and this is not specified by a subclass implementation, the regular enumNames() will be assumed to apply for both move() and value(). moveEnumNames() will return enumNames(). */
 	void setMoveEnumStates(const QStringList& enumStateNames) { if(moveEnumNames_ == enumStateNames) return; moveEnumNames_ = enumStateNames; emit enumChanged(); }
 
+	/// Sets the move start value.
+	void setMoveStart(double newValue);
+	/// Updates the move start value.
+	virtual void updateMoveStart();
```

Comment:
I didn't get the purpose of `updateMoveState()`, `updateMoveValue()` and `updateMoveEnd()`. those functions are called in the `move()` function of the PVControl. I think we should only need to `updateMoveValue()`. The other two are set before we perform the move and won't be changed when we are moving.